sql/core/src/test/resources/sql-tests/inputs/comments.sql

```diff
@@ -1,12 +1,5 @@
 -- Test comments.
-
--- the first case of bracketed comment
---QUERY-DELIMITER-START
-/* This is the first example of bracketed comment.
-SELECT 'ommented out content' AS first;
-*/
-SELECT 'selected content' AS first;
---QUERY-DELIMITER-END
+--IMPORT nested-comments.sql
```

Comment:
What did this means?
```
Golden files can't display well.
```

Reply:
I updated the description to `Golden files can't display the bracketed comments in imported test cases`.

Reply:
Also, could you put an output example before/after this PR in the PR description?

Reply:
OK. I have put the output example